### PR TITLE
Microverse Capstone Project

### PR DESCRIPTION
**Pull request by tacettinkustu**
In this project, I created a website for announcement of a conference. The website was created using HTML, CSS and JavaScript to meet the requirements defined within the scope of the Microverse Capstone project. For this purpose, I follow these steps:

1. Set up the repository and tools.
2. Choose the topic for your website.
3. Choose images, text, icons, fonts.
4. Create the 2 pages for mobile.
5. Deploy the project.
6. Adapt the 2 pages to desktop.
7. Implement the user interactions: link, mobile menu.
8. Deploy the project.
9. Implement the section "Featured speakers" with dynamic HTML.
10. Deploy the project and check to make sure everything works as planned.
11. Record a video describing your project.
12. Create a good README and pull request description.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+          src="./utils/images/—Pngtree—line-decorative-design-pattern_3634817.png"
+          alt="Background image"
+          class="bg-header-image-rotate"
+        />
+        <div class="main-content">
+          <div class="main-content-headers">
+            <h2 class="dark-orange">"Hello! Sharing world"</h2>
+            <h1 class="dark-orange">
+              YTU ENTREPRENEURSHIP and TECHNOLOGY CONFERENCE
+            </h1>
```
Comment (Shaher-11) [inline about.html]: Change the color here  and in the home page as shown in the original design below

#### Original Design
right1

#### Your Project
wrong1

File: about.html [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    />
+    <!-- Mobile CSS -->
+    <link rel="stylesheet" href="css/global.css" />
+    <link rel="stylesheet" href="css/main.css" />
+    <!-- Desktop CSS -->
+    <link rel="stylesheet" media="(min-width:768px)" href="css/desktop.css" />
+  </head>
+  <body class="flex-column-center">
+    <header class="bg-light-grey flex-column-center">
+      <nav>
```
Comment (Shaher-11) [inline about.html:50]: Add a div here to hold the top navbar like the one below

right

File: index.html [hunk 3]
```diff
@@ -29,9 +36,221 @@
...
+          />
+        </li>
+        <li class="list-item-container mozilla">
+          <img
+            class="list-item"
+            src="./utils/images/amazon-lightsail.svg"
+            alt="mozilla"
+          />
+        </li>
+      </ul>
```
Comment (Shaher-11) [inline index.html:241]: Reduce the size of the images in this section and align them as shown below
#### Original Design
right2

#### Your Project 
wrong2

File: index.html [hunk 4]
```diff
@@ -29,9 +36,221 @@
...
+              2021.06.28(MON) ~ 30(WED)
+            </h3>
+            <p class="location light-black heading-small">
+              @Hall of Yildiz Technical University, Istanbul, Turkey
+            </p>
+          </div>
+        </div>
+      </div>
+    </header>
+    <section class="main-program flex-column-center" id="main-program">
```
Comment (Shaher-11) [inline index.html:125]: Style the h3 Elements in this section with the orange color as the original design
right3

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @tacettinkustu,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [x] Check the comments below:point_down:


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project . If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: about.html (hunk 1 above)
Comment (tacettinkustu) [reply about.html]: I changed the color of these texts.

File: about.html (hunk 2 above)
Comment (tacettinkustu) [reply about.html:50]: I created this black section at the upmost of my desktop version

File: index.html (hunk 3 above)
Comment (tacettinkustu) [reply index.html:241]: I reduced the sizes of my partner logos and take them all into one row.

File: index.html (hunk 4 above)
Comment (tacettinkustu) [reply index.html:125]: I change the colors of these titles.

**Comment by tacettinkustu** [comment]
> #### Hello @tacettinkustu,
> ### Great job so far!
> You have been doing an incredible job on the project so far👏👏
> The project is very well structured😃😃
> However, we still need to make a few changes to get the project approved💪
> 
> ## Required Changes ♻️
> * [x]  Check the comments below👇
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project . If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> Happy Coding!👏👏👏

Thanks for your insightful reviews.

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @tacettinkustu , 

You did an excellent job with making all the changes requested by the previous TSE. 

Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tacettinkustu** [comment]
> # Approved ✔️
> Hi @tacettinkustu ,
> 
> You did an excellent job with making all the changes requested by the previous TSE.
> 
> Your project is complete!
> Congratulations! 🎉
> 
> You can merge this pull request and continue with your journey 👋
> 
> Happy Hacking! 💻
> 
> You can reply to this conversation if you have any questions or need elaboration. 👋
> Sinan Sevgi
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks for your insightful reviews.

**Pull request merged**